packages/cli/src/utils/npm-utils.ts

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import type { BinaryLike } from 'crypto';
+import crypto from 'crypto';
+import { UnexpectedError } from 'n8n-workflow';
+
+export async function verifyIntegrity(
+	packageName: string,
+	version: string,
+	registryUrl: string,
+	expectedIntegrity: string,
+) {
+	try {
+		registryUrl = `${registryUrl}/${packageName.replace('/', '%2F')}`;
```

Comment:
```suggestion
		registryUrl = new URL(encodeURIComponent(packageName), registryUrl).toString();
```

Reply:
Would also prefer to make a new variable instead of reassigning registryUrl, could also include `version` already